src/bundle/Resources/views/content/locationview.html.twig

```diff
@@ -66,7 +66,12 @@
                     <div class="container px-5">
                         <section class="ez-fieldgroup">
                             <h2 class="ez-fieldgroup__name">{{ 'content.view.subitems'|trans|desc('Sub-items') }}</h2>
-                            <div class="ez-sil" data-location="{{ location.id }}" data-items="{{ subitems_module.items }}" data-content-types="{{ subitems_module.content_type_info_list }}"></div>
+                            <div class="ez-sil" 
+                                data-location="{{ location.id }}" 
+                                data-items="{{ subitems_module.items }}" 
+                                data-content-types="{{ subitems_module.content_type_info_list }}"
+                                data-udw-config="{{ ez_udw_config('single', {}) }}"
```

Comment:
`single_container` not `single`
And I was thinking about the name of a data-attribute. What if we would need to pass another configuration of UDW in other button?
I think we need to call it more precise like `data-move-udw-config`.

Reply:
Fixed: https://github.com/ezsystems/ezplatform-admin-ui/pull/702/commits/19163722885b5a1462e1886515e691f6143ef6eb